src/routes/extra_functionality_movie.py

```
@@ -0,0 +1,460 @@
+        await db.commit()
+        return MessageResponse(message="Reaction added")
+
+
+@router.get(
+    "/movies/{movie_id}/likes-stats",
+    response_model=dict,
+    summary="Get movie likes/dislikes statistics",
+    description="Returns the count of likes and dislikes for a movie.",
+)
+async def get_movie_likes_stats(
+    movie_id: int, db: AsyncSession = Depends(get_db)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)